### ISSUE TITLE
Server is crashing for /:userId/recipes

### ISSUE THREAD

**Issue by udeyrishi** [SmartFoodTracker/backend#6]
Logs:

```
2017-02-21T05:35:55.507505+00:00 app[web.1]: /app/dist/routes/recipes.js:102
2017-02-21T05:35:55.507514+00:00 app[web.1]: 				data: payloadData,
2017-02-21T05:35:55.507516+00:00 app[web.1]: 				      ^
2017-02-21T05:35:55.507517+00:00 app[web.1]: 
2017-02-21T05:35:55.507518+00:00 app[web.1]: ReferenceError: payloadData is not defined
2017-02-21T05:35:55.507519+00:00 app[web.1]:     at Request._callback (/app/dist/routes/recipes.js:102:11)
2017-02-21T05:35:55.507520+00:00 app[web.1]:     at Request.self.callback (/app/node_modules/request/request.js:186:22)
2017-02-21T05:35:55.507520+00:00 app[web.1]:     at emitTwo (events.js:106:13)
2017-02-21T05:35:55.507521+00:00 app[web.1]:     at Request.emit (events.js:191:7)
2017-02-21T05:35:55.507521+00:00 app[web.1]:     at Request.<anonymous> (/app/node_modules/request/request.js:1081:10)
2017-02-21T05:35:55.507522+00:00 app[web.1]:     at emitOne (events.js:96:13)
2017-02-21T05:35:55.507523+00:00 app[web.1]:     at Request.emit (events.js:188:7)
2017-02-21T05:35:55.507524+00:00 app[web.1]:     at IncomingMessage.<anonymous> (/app/node_modules/request/request.js:1001:12)
2017-02-21T05:35:55.507525+00:00 app[web.1]:     at IncomingMessage.g (events.js:291:16)
2017-02-21T05:35:55.507525+00:00 app[web.1]:     at emitNone (events.js:91:20)
2017-02-21T05:35:55.507526+00:00 app[web.1]:     at IncomingMessage.emit (events.js:185:7)
2017-02-21T05:35:55.507527+00:00 app[web.1]:     at endReadableNT (_stream_readable.js:974:12)
2017-02-21T05:35:55.507527+00:00 app[web.1]:     at _combinedTickCallback (internal/process/next_tick.js:74:11)
2017-02-21T05:35:55.507528+00:00 app[web.1]:     at process._tickCallback (internal/process/next_tick.js:98:9)
2017-02-21T05:35:55.525311+00:00 app[web.1]: 
2017-02-21T05:35:55.538206+00:00 app[web.1]: npm ERR! Linux 3.13.0-105-generic
2017-02-21T05:35:55.538558+00:00 app[web.1]: npm ERR! argv "/app/.heroku/node/bin/node" "/app/.heroku/node/bin/npm" "run" "serve"
2017-02-21T05:35:55.538838+00:00 app[web.1]: npm ERR! node v6.9.5
2017-02-21T05:35:55.539043+00:00 app[web.1]: npm ERR! npm  v3.10.10
2017-02-21T05:35:55.539262+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2017-02-21T05:35:55.539440+00:00 app[web.1]: npm ERR! food-fit@1.0.0 serve: `NODE_ENV=production node dist/index.js`
2017-02-21T05:35:55.539628+00:00 app[web.1]: npm ERR! Exit status 1
2017-02-21T05:35:55.539851+00:00 app[web.1]: npm ERR! 
2017-02-21T05:35:55.540049+00:00 app[web.1]: npm ERR! Failed at the food-fit@1.0.0 serve script 'NODE_ENV=production node dist/index.js'.
2017-02-21T05:35:55.540239+00:00 app[web.1]: npm ERR! Make sure you have the latest version of node.js and npm installed.
2017-02-21T05:35:55.540423+00:00 app[web.1]: npm ERR! If you do, this is most likely a problem with the food-fit package,
2017-02-21T05:35:55.540602+00:00 app[web.1]: npm ERR! not with npm itself.
2017-02-21T05:35:55.541038+00:00 app[web.1]: npm ERR! Tell the author that this fails on your system:
2017-02-21T05:35:55.541522+00:00 app[web.1]: npm ERR!     NODE_ENV=production node dist/index.js
2017-02-21T05:35:55.541723+00:00 app[web.1]: npm ERR! You can get information on how to open an issue for this project with:
2017-02-21T05:35:55.541918+00:00 app[web.1]: npm ERR!     npm bugs food-fit
2017-02-21T05:35:55.542328+00:00 app[web.1]: npm ERR! Or if that isn't available, you can get their info via:
2017-02-21T05:35:55.542516+00:00 app[web.1]: npm ERR!     npm owner ls food-fit
2017-02-21T05:35:55.542731+00:00 app[web.1]: npm ERR! There is likely additional logging output above.
2017-02-21T05:35:55.568403+00:00 app[web.1]: 
2017-02-21T05:35:55.568581+00:00 app[web.1]: npm ERR! Please include the following file with any support request:
2017-02-21T05:35:55.568651+00:00 app[web.1]: npm ERR!     /app/npm-debug.log
2017-02-21T05:35:55.521828+00:00 heroku[router]: at=error code=H13 desc="Connection closed without response" method=GET path="/1/recipes?page=0" host=food-fit.herokuapp.com request_id=da99ccea-6b1d-43bb-8c47-749f2fc344bf fwd="104.205.129.56" dyno=web.1 connect=0ms service=527ms status=503 bytes=0
2017-02-21T05:35:55.651308+00:00 heroku[web.1]: State changed from up to crashed
2017-02-21T05:35:55.644055+00:00 heroku[web.1]: Process exited with status 1
2017-02-21T05:35:55.541723+00:00 app[web.1]: npm ERR! You can get information on how to open an issue for this project with:
2017-02-21T05:35:55.541918+00:00 app[web.1]: npm ERR!     npm bugs food-fit
2017-02-21T05:35:55.542328+00:00 app[web.1]: npm ERR! Or if that isn't available, you can get their info via:
2017-02-21T05:35:55.542516+00:00 app[web.1]: npm ERR!     npm owner ls food-fit
2017-02-21T05:35:55.542731+00:00 app[web.1]: npm ERR! There is likely additional logging output above.
2017-02-21T05:35:55.568403+00:00 app[web.1]: 
2017-02-21T05:35:55.568581+00:00 app[web.1]: npm ERR! Please include the following file with any support request:
2017-02-21T05:35:55.568651+00:00 app[web.1]: npm ERR!     /app/npm-debug.log
2017-02-21T05:35:55.521828+00:00 heroku[router]: at=error code=H13 desc="Connection closed without response" method=GET path="/1/recipes?page=0" host=food-fit.herokuapp.com request_id=da99ccea-6b1d-43bb-8c47-749f2fc344bf fwd="104.205.129.56" dyno=web.1 connect=0ms service=527ms status=503 bytes=0
2017-02-21T05:35:55.651308+00:00 heroku[web.1]: State changed from up to crashed
2017-02-21T05:35:55.644055+00:00 heroku[web.1]: Process exited with status 1
```

**Comment by udeyrishi** [comment]
Seems to be fixed by cd2133855ef91a1c29483f5ce6679b7f69a510b2

**Comment by ahoskins** [comment]
This was silly. Must have been tired when making this initial error handling.
